crates/apub/src/protocol/objects/group.rs

```diff
@@ -50,10 +51,16 @@ pub struct Group {
 
   /// title
   pub(crate) name: Option<String>,
-  pub(crate) summary: Option<String>,
+  // sidebar
+  #[serde(deserialize_with = "deserialize_skip_error", default)]
+  pub(crate) content: Option<String>,
   #[serde(deserialize_with = "deserialize_skip_error", default)]
   pub(crate) source: Option<Source>,
   #[serde(deserialize_with = "deserialize_skip_error", default)]
+  pub(crate) media_type: Option<MediaTypeHtml>,
+  // short instance description
+  #[serde(deserialize_with = "deserialize_skip_error", default)]
+  pub(crate) summary: Option<String>,
   pub(crate) icon: Option<ImageObject>,
```

Comment:
You accidentally removed the line `#[serde(deserialize_with = "deserialize_skip_error", default)]` for icon, thats why tests are failing.

Reply:
Aaahh dang, thx.